crates/http/src/routes.rs

```diff
@@ -235,25 +219,28 @@ impl fmt::Display for ParsedRoute {
 }
 
 /// A routing match for a URL.
-pub struct RouteMatch {
-    route_handler: RouteHandler,
-    named_wildcards: HashMap<String, String>,
+pub struct RouteMatch<'router, 'path> {
+    /// The route handler that matched the path.
+    route_handler: Cow<'router, RouteHandler>,
+    /// An optional best match for the path.
+    best_match: Option<(&'path str, routefinder::Match<'router, 'path, RouteHandler>)>,
```

Comment:
How would you feel about replacing the tuple with a struct?  I assume this would be no more expensive, but it could make it more self-documenting.

If you're not keen, maybe tweak the doc comment to say what the tuple elements are?

Reply:
I tweaked the code to hopefully be a bit more self documenting: https://github.com/spinframework/spin/pull/3107/commits/ee1560633bccd164646e11619158c53a51e5731b

Let me know what you think.